client-hc/src/main/java/com/graphhopper/api/GraphHopperWeb.java

```diff
@@ -236,6 +236,11 @@ private Request createPostRequest(GHRequest ghRequest) {
         String url = tmpServiceURL + "?";
         if (!Helper.isEmpty(key))
             url += "key=" + key;
+        if (!Helper.isEmpty(ghRequest.getProfile())) {
+            if (!url.endsWith("?"))
+                url += "&";
+            url += "profile=" + ghRequest.getProfile();
+        }
```

Comment:
Ok at some point I thought it would be useful to have `profile` as url parameter, so one can quickly decide (without looking at the request body) what kind of profile was requested (e.g. when filtering requests).